lca-cli/postpivot/postpivot.go

```diff
@@ -478,6 +479,47 @@ func (p *PostPivot) setNodeIPIfNotProvided(ctx context.Context,
 	return nil
 }
 
+func (p *PostPivot) setNodeIPHint(ctx context.Context, nodeIp, nodeIPConfFile string) error {
+	if nodeIp == "" {
+		// TODO: maybe better to delete nodeip hint file in this case?
+		p.log.Info("Node ip was not provided, skipping node ip hint changes")
+		return nil
+	}
+	var hintSubnet net.IP
```

Comment:
I wonder if we care about the subnet?
We know the IP, why not just write the IP with CIDR 32?
192.168.127.10/32, won't this work?

Reply:
nope, nodeip-hint expects to get like 192.168.127.0 , first ip in subnet